src/components/AssetHeader/AssetActions.tsx

```diff
@@ -11,13 +11,12 @@ import { selectAssetById } from 'state/slices/selectors'
 import { useAppSelector } from 'state/store'
 
 type AssetActionProps = {
-  isLoaded: boolean
   assetId: AssetId
   accountId?: AccountSpecifier
   cryptoBalance: string
 }
 
-export const AssetActions = ({ isLoaded, assetId, accountId, cryptoBalance }: AssetActionProps) => {
+export const AssetActions: React.FC<AssetActionProps> = ({ assetId, accountId, cryptoBalance }) => {
```

Comment:
I think generally we want to explicitly import UMD global variables, as it can lead to a runtime error if the library isn't included implicitly.

Less important for `React`, because if we don't have that globally we have bigger issues - but good to enforce convention, I might lint it at some point.

Reply:
yeah, happy to add this as a lint rule, but agree in practicality if we don't have React we'll prob know pretty quickly